frontend/src/components/table_view.tsx

```
@@ -0,0 +1,132 @@
+
+    if (!id) {
+      reset_to_main();
+      return;
+    }
+
+    dispatch(update_table({ id: id }));
+  };
+
+  // We want to make coherent tables, seperate the non-nested from the nested
+  const [non_nested, nested]: [ColumnSchema[], ColumnSchema[]] = [[], []];
+  for (let i = 0; i < data.children!.length; i++) {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)